gui/src-tauri/src/main.rs

```diff
@@ -31,6 +32,10 @@ static POSSIBLE_TITLES: &[&str] = &[
 	"never gonna let you down",
 	"uwu sowwy",
 ];
+
+#[derive(Default)]
```

Comment:
Add a doc comment explaining what this struct is for.

Reply:
added